cfy_cluster_setup/main.py

```diff
@@ -291,13 +298,57 @@ def generate_config(output_path):
                 output_path)
 
 
+def _check_path(config, key):
+    path = config.get(key)
+    if (not path) or (not exists(expanduser(path))):
+        raise ValidationError('Path {0} does not exist'.format(path))
```

Comment:
One thing that might make for a better UX is to collect all errors and then raise at the end with the full list, rather than them having to run the validation $number_of_errors times.

Reply:
Makes sense, I'll add it.